app/Providers/AppServiceProvider.php

```diff
@@ -37,6 +42,11 @@ public function boot()
         Relation::morphMap([
             'applicant' => Applicant::class,
             'application' => JobApplication::class,
+            'work' => ExperienceWork::class,
+            'personal' => ExperiencePersonal::class,
+            'education' => ExperienceEducation::class,
+            'award' => ExperienceAward::class,
+            'community' => ExperienceCommunity::class,
```

Comment:
Let's change these strings to 'experience_work', 'experience_personal', etc, to make the connection nice and explicit.